.github/workflows/test.yml

```diff
@@ -7,10 +7,10 @@ on:
       - master
 jobs:
   test:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [12,13,14,15,16,17]
+        node-version: [12,13,14,15,16,17,18,19]
```

Comment:
Node 13, 15 & 17 never made it to LTS and are EOL now. Is it still worth running tests on them?

Node 12 LTS is EOL, but I can understand why you'd want to keep testing against it